docs/core-concepts/bot-sequences/getting-started-with-bot-sequences-and-segments.mdx

```diff
@@ -2,8 +2,58 @@
 sidebar_label: 'Getting Started With Bot Sequences and Segments'
 ---
 
+import SequenceCard from './img/sequence_card.png';
+import SequenceDeleteDialog from './img/sequence_delete.png';
+import CreateSequenceButton from './img/create_sequence_button.png';
+import ReloadIcon from './img/reload.png';
+
 # Getting Started With Bot Sequences and Segments
 
-import Placeholder from '../../partials/_placeholder.mdx';
+A **Bot Sequence** is a series of automated actions that can be used to assert certain behaviours or outcomes within your game. Sequences are composed of individual actions, called Segments, which are executed in order.
+
+![Bot Sequence List in RGOverlayCanvas](img/sequences_list.png)
+
+## Starting A Bot Sequence
+
+After [adding the RGOverlayCanvas prefab to your scene](../the-in-game-overlay.mdx), click on the small Williward face icon to open the overlay. With the overlay opened, you can see the list of Bot Sequences that are available to you. Each Sequence is represented by a card, with three possible actions:
+- **Play**: Start the Bot Sequence, beginning with its first Segment
+- **Edit**: Open the Bot Sequence in the Sequence Editor (see the next section)
+- **Delete**: Delete the Bot Sequence file from your project (see the next section)
+
+
+<div style={{width: '100%'}}>
+<img src={SequenceCard} alt="Sequence Card" style={{display: 'block', width: '300px', margin: '0 auto'}} />
+</div>
+
+## Creating and Modifying Bot Sequences
+
+### Within Unity
+
+Click the "Create a new Sequence" button, with the overlay open, to create a new Bot Sequence in our Sequence Editor.
+
+<div style={{width: '100%'}}>
+<img src={CreateSequenceButton} alt="Create Sequence Card" style={{display: 'block', width: '300px', margin: '0 auto'}} />
+</div>
+
+<br />
+
+Within the Sequence Editor, you can add, remove, and reorder Segments. The available Segments can be searched for by text, and are added to the Sequence by dragging them from the list on the right to the Sequence in the middle of the editor. When a Sequence is played, it will execute its Segments from top to bottom.
```

Comment:
This page in its current state introduces segments without explaining what they are

Reply:
@abeizer this ticket specifically asks for overlay docs - the details about segments and sequences will be covered by a different ticket. I am breaking down my docs ticket into more pieces to make this more apparent!